ext/QVD-HKD/Makefile.PL

```diff
@@ -35,7 +35,8 @@ WriteMakefile(
         'Linux::Proc::Net::UDP' => 0,
         'Method::WeakCallback' => 0,
         'URI::Escape' => 0,
-        'BSD::Resource' => '1.2910'
+        'BSD::Resource' => '1.2910',
+	'Filesys::ZFS' => '1.5'
```

Comment:
It seems that Filesys::ZFS is just a thin wrapper around `zpool` and `zfs` commands and you are not actually using more than its `_run` method. So, I think we can remove its usage completely from the module.